galite-core/src/main/kotlin/org/kopi/galite/ui/vaadin/actor/Actor.kt

```diff
@@ -51,8 +52,7 @@ open class Actor(val caption: String?,
     super.setText(caption)
 
     if (icon != null) {
-      val img = Image()
-      img.src = icon
+      val img = Icon(icon)
```

Comment:
can we change the size of the icon for more readability : 
     ` img.setSize("25px")`